### DISCUSSION TITLE
Shift the rotated image up/down or lef/right

### DISCUSSION THREAD

**Discussion by Rudi1986** [jomjol/AI-on-the-edge-device#2892] [General]
### The Feature

I aligned my cam in a way that reflections don't match with the digits. In my case the rotation angle is at 231,3°.
When i view the raw image (stream) my meter is dead center. But when the rotation is done one of the analog ROIs (0.001) is slightly cropped at the buttom. This leads to an error while further processing the image.
So is there a way to shift the image bevore rotating and cropping it?

Some example:
![canvas-2](https://github.com/jomjol/AI-on-the-edge-device/assets/42764579/4b372a3f-9e16-4614-b0d0-9030ef2bdd3c)
![canvas](https://github.com/jomjol/AI-on-the-edge-device/assets/42764579/cf501049-10bc-49e5-b9d7-66acac195b60)
![main_ana3](https://github.com/jomjol/AI-on-the-edge-device/assets/42764579/71c17b0c-8b30-4a98-9560-ea15a59f614e)

**Comment by Slider0007** [comment]
Your ROI looks not processed properly. ROI area shall not be out of image area (every pixel counts) to avoid irratic system behaviour. Just make the ROI smaller...

BTW: Alignment marker pattern algo is already shifting and rotating to find best match.
Note: The alignment algo can even be switched off (expert parameter -> OFF): Then only pure rotation correction defined in reference image screen is getting applied.
